### DISCUSSION TITLE
0.17 / release cadence?

### DISCUSSION THREAD

**Discussion by NJAldwin** [fdehau/tui-rs#575] [Q&A] [answered]
I was recently exploring creating a new TUI with this library.  I started out with the 0.16 examples, then noticed that the `master` examples are a lot more straightforward due to the use of crossterm.  I'd like to write my app like these examples, but there hasn't been a `tui-rs` release since Aug (0.16), and thus no release with the `crossterm` feature as a default.

For development, I can always point my app at the git repo directly, but I would prefer to use a tagged release, especially for a significant API change such as switching to crossterm.

Thus, I'm wondering: what's the plan for releasing `tui-rs` v0.17?

Please don't take this as a demand -- no rush, I'm just wondering what to expect for planning purposes.  Love the library!

**Comment by fdehau** [comment, marked as answer]
Hey 👋 Sorry for the slow response. You are right, a new release ought to be released by now but I have very little time to dedicate to `tui` these days 😬.

I've just published [v0.17.0](https://github.com/fdehau/tui-rs/releases/tag/v0.17.0).
